openfe/setup/atom_mapping/perses_mapper.py

```diff
@@ -0,0 +1,87 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe
+
+"""
+The MCS class from Perses shamelessly wrapped and used here to match our API.
+
+"""
+
+from openmm import unit
+
+# Test if Openeye license present! on class construction.
+from openff.toolkit.utils.exceptions import LicenseError
+from openff.toolkit.utils import toolkits
+
+from perses.rjmc.atom_mapping import AtomMapper, InvalidMappingException
```

Comment:
we'll have to make perses optional here

Reply:
so a pattern to look up is how openff toolkit has a `requires` decorator that you can label functions/classes with.

Reply:
suggestion: openfe/utils/integration_tools.py

one could probably also pack this into a decorator. would that work for you?

Reply:
ok I will try the require dekorator.